docs/datagrid/columns.md

```diff
@@ -46,11 +46,11 @@ Note that most of the options in `columnDefaults` are mirrored 1:1 in the `colum
 }
 ```
 
-The `columns[].header` option can be used to configure the header for individual columns. If needed you can set defaults for all columns in `columnDefaults.header`.
+The `columns[].header` option can be used to configure the header for individual columns. If needed, you can set defaults for all columns in `columnDefaults.header`.
 
-Note that `className` and `format` supports templating as described in [Templating](https://www.highcharts.com/docs/chart-concepts/templating), and `{id}` references the key in the DataTable.
+Note that `className` and `format` support templating as described in [Templating](https://www.highcharts.com/docs/chart-concepts/templating), and `{id}` references the key in the DataTable.
 
-If you need more advanced formatting that is not supported through templating use the `formatter` callback function instead. As in Highcharts Core, we always recommend `format` if possible. [Read more here...](https://www.highcharts.com/docs/chart-concepts/labels-and-string-formatting#formatter-callbacks)
+Suppose you need more advanced formatting that is not supported through templating. Use the `formatted` callback function instead. As in Highcharts Core, we always recommend `format` if possible. [Read more here...](https://www.highcharts.com/docs/chart-concepts/labels-and-string-formatting#formatter-callbacks)
```

Comment:
No such thing as `formatted` callback, and I don't agree that suggested change is better.